.ci/blc-website-preview

```diff
@@ -43,26 +47,26 @@ GRN=$'\033[1;32m'
 BLD=$'\033[1m'
 END=$'\033[0m'
 if [[ $num_complaints -eq 0 ]]; then
-	printf "%s======================= 0 broken-link-checker complaints =======================%s\n" "$GRN$BLD" "$END"
+	printf "%s======================= 0 broken-link-checker complaints for %s =======================%s\n" "$GRN$BLD" "$website_branch" "$END"
```

Comment:
I'd vote for leaving the branch name in, FTR. No harm.

Reply:
It always uses master now, so it's not super useful to just always say master.